nebula-logger/core/main/log-management/classes/LogHandler.cls

```diff
@@ -27,6 +28,7 @@ public without sharing class LogHandler extends LoggerSObjectHandler {
 
     protected override void executeBeforeInsert(List<SObject> triggerNew) {
         this.logs = (List<Log__c>) triggerNew;
+        this.logScenariosById = queryLogScenarios(this.logs);
```

Comment:
missing the `this` prefix here and in the below usage of this method, not sure if that was intentional or not